Cargo.toml

```diff
@@ -8,8 +8,8 @@ authors = ["Bernhard Schuster <bernhard@ahoi.io>",
 
 [dependencies]
 # cuticula = { path = "../cuticula", default-features = false }
-juice = { path = "../juice", default-features = false, version = "0.2.2" }
-coaster = { path = "../coaster", default-features = false, version = "0.1.0" }
+juice = { default-features = false, version = "0.2.2" }
+coaster = { default-features = false, version = "0.1.0" }
 
```

Comment:
this pretty much cuts all ties to your local checkouts, hence it either means that something got borked between now and the `creates.io` version or something you locally checkout is borked.